flink-libraries/flink-table/pom.xml

```diff
@@ -154,6 +154,10 @@ under the License.
 				<artifactId>maven-shade-plugin</artifactId>
 				<executions>
 					<execution>
+						<phase>package</phase>
+						<goals>
+							<goal>shade</goal>
```

Comment:
Yes. #2673 is blocked.  Without this, my IDEA Editor couldn't resolve the shade configuration, so I introduce it in this PR. Should we remove this ? 

![image](https://cloud.githubusercontent.com/assets/5378924/20417319/757a9128-ad7f-11e6-99d7-1dc603c425a8.png)